### PR TITLE
DOCS-3440 Adds Generate Metrics From RUM Events Beta Documentation

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds Generate Metrics beta documentation page, screenshots, and TOC updates.

### Motivation

DOCS-3440 

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/generate_metrics/_index.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+    - Select `*` to generate a count of all RUM events that match your search query. 
+    - Optionally, enter an event attribute such as `@action.target` to aggregate a numeric value and create corresponding, aggregated metrics such as `count`, `min`, `max`, `sum`, and `avg`. 
+
+   If the RUM attribute facet is a measure, the metric value is the RUM attribute value.
+
+4. Select a path to group by from the dropdown menu next to **group by**. The metric tag name is the original attribute or tag name without the `@`. By default, metrics generated from RUM events do not contain tags unless they are explicitly added. You can use an attribute or tag dimension that exists in your RUM events such as `@error.source` or `env` to create metric tags. 
+   
+   <div class="alert alert-warning">Custom RUM metrics are considered as <a href="/metrics/custom_metrics/">custom metrics</a>. Datadog recommends avoiding grouping by unbounded or extremely high cardinality attributes such as timestamps, user IDs, request IDs, and session IDs. For more information, see <a href="/data_security/logs/">Log Management Data Security</a>.
```
Comment (urseberry) [inline content/en/real_user_monitoring/generate_metrics/_index.md:55]:
```suggestion
   <div class="alert alert-warning">Custom RUM metrics are considered to be <a href="/metrics/custom_metrics/">custom metrics</a>. Datadog recommends avoiding grouping by unbounded or extremely high cardinality attributes such as timestamps, user IDs, request IDs, and session IDs. For more information, see <a href="/data_security/logs/">Log Management Data Security</a>.
```

File: content/en/real_user_monitoring/generate_metrics/_index.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+- Aggregation groups: Update tags or manage the cardinality of generated metrics.
+- Percentile selection: Click the **Calculate percentiles** toggle to remove or generate percentile metrics.
+
+Because you cannot rename a metric, Datadog recommends creating another metric.
+
+### Delete a custom RUM metric
+
+In order to stop the computing of data points from your metric (and billing), hover over a metric and click the **Delete** icon to the right hand corner. 
+
+## Use a custom RUM metric
```
Comment (urseberry) [inline content/en/real_user_monitoring/generate_metrics/_index.md:83]: This is a matter of opinion, but "Usage" might be a simpler heading for this section.

File: content/en/real_user_monitoring/generate_metrics/_index.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+You can use custom RUM metrics to:
```
Comment (urseberry) [inline content/en/real_user_monitoring/generate_metrics/_index.md:85]:
```suggestion
You can use custom RUM metrics to do the following:
```

**Review by urseberry — APPROVED** [review]
Great organization and use of links! Just a few copy nits from me.

**Comment by amina196** [comment]
just need to change the screenshot for this one: https://a.cl.ly/KouApGk2
then you can consider the PR approved!

**Pull request merged**